docs/gnmi.md

```diff
@@ -56,6 +61,8 @@ gnmi_cli -get -address onos-config:5150 \
 
 ### List all device names (targets)
 A useful way to retrieve all stored device names is with the command:
+
+[gnmi](https://github.com/onosproject/onos-config/tree/master/gnmi_cli/get.alldevices.gnmi)
```

Comment:
What does it do? I checked markdown file in your local but that path doesn't exist.

Reply:
but it will when it's deployed - it will link to github